extensions/amp-base-carousel/0.1/test-e2e/test-max-swipe.js

```diff
@@ -50,26 +43,21 @@ describes.endtoend('AMP carousel max-swipe', {
   });
 
   describe('looping', () => {
-    beforeEach(async() => {
-      await controller.navigateTo(
-          'http://localhost:8000/test/manual/amp-base-carousel/max-swipe-one.amp.html');
-    });
-
     it('should only show one slide on each side initially', async() => {
       const el = await getScrollingElement(controller);
       const slides = await getSlides(controller);
 
-      await expect(prop(el, 'scrollLeft')).to.equal(pageWidth);
+      await controller.waitForElementProperty(el, 'scrollLeft', pageWidth);
```

Comment:
Again, I don't think we should have separate functions here, we should use standard `expects`, so that we can use other expect types (like for example, checking that it is with some range of values or close to a certain value).